src/index.js

```diff
@@ -0,0 +1,148 @@
+const state = {
+
+};
+
+const setUpTemperature = () => {
+  let classType;
+  let gardenType;
+  // let temperature = state.temp
+
+  if (state.temp >= 80) {
+    classType = "red";
+    gardenType = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+  } else if ((state.temp >= 70) & (state.temp <= 79)) {
+    classType = "orange";
+    gardenType = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+  } else if ((state.temp >= 60) & (state.temp <= 69)) {
+    classType = "yellow";
+    gardenType = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+  } else if ((state.temp >= 50) & (state.temp <= 59)) {
+    classType = "green";
+    gardenType = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  } else {
+    classType = "teal";
+    gardenType = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  }
+
+  let temp = document.querySelector("#temp");
+  temp.setAttribute("class", classType);
+  temp.textContent = state.temp;
+
+  let garden = document.querySelector("#garden");
+  garden.textContent = gardenType;
+  garden.setAttribute("class", classType);
+};
+
+function Up() {
+  let temp = document.querySelector("#temp");
+  state.temp = state.temp + 1;
+  temp.textContent = state.temp;
+  setUpTemperature();
+}
+
+function Down() {
+  const temp = document.getElementById("temp");
+  state.temp = state.temp - 1;
+  temp.textContent = state.temp;
+  setUpTemperature();
+}
+
+// update city name and h2 values
+function City() {
+  const cityName = document.getElementById("cityname").value;
+  const cityHeader = document.querySelector("h2");
+  cityHeader.textContent = cityName;
+  state.cityName = cityName;
+}
+
+function clearInput() {
+  const cityName = document.getElementById("cityname");
+  const cityHeader = document.querySelector("h2");
+  if (cityName.value != "") {
+    cityName.value = "";
+    cityHeader.textContent = "";
+  }
+}
+
+mySelect = document.getElementById("sky");
+mySelect.onchange = function selectSky() {
+  selectedSky = document.getElementById("sky").value
+  pElement = document.getElementById("sky-emoji")
+  if (selectedSky == "rainy") {
+    pElement.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+  } else if (selectedSky == "sunny") {
+    pElement.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+  } else if (selectedSky == "snowy") {
+    pElement.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+  } else if (selectedSky == "cloudy") {
+    pElement.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+  }
+}
+
+function findRealTimeTemp() {
+  findLatitudeAndLongitude();
+}
+
+const findLatitudeAndLongitude = (query) => {
```

Comment:
```suggestion
const findLatitudeAndLongitude = () => {
```
It doesn't look like the `query` parameter gets used anywhere so you can get rid of it